config.ini

```diff
@@ -5,16 +5,24 @@ TERM = xterm
 # to raise the rate limit for GitHub API
 NVCHECKER_GITHUB_TOKEN = 
 
+[repository]
+email = repo@archlinuxcn.org
+repodir = ~/archgitrepo
+destdir = ~/repo
+
 [lilac]
 name = lilac
 email = lilac@build.archlinuxcn.org
 master = lilydjwg <lilydjwg@gmail.com>
 log_to_file = on
 git_push = false
 
-[repository]
-email = repo@archlinuxcn.org
-repodir = ~/archgitrepo
-destdir = ~/repo
+[smtp]
+host = localhost
+# leave it to empty to use default port
+port = 
+use_ssl = no
+username = 
+password = 
```

Comment:
Add configurations to `config.ini.sample` too, leaving `config.ini` for use on build machine.